test/boost/sstable_directory_test.cc

```diff
@@ -95,7 +96,18 @@ make_sstable_for_all_shards(replica::database& db, replica::table& table, fs::pa
 }
 
 sstables::shared_sstable new_sstable(sstables::test_env& env, fs::path dir, int64_t gen) {
-    return env.make_sstable(test_table_schema(), dir.native(), generation_from_value(gen));
+    testlog.debug("new_sstable: dir={} gen={}", dir, gen);
+    return env.make_sstable(test_table_schema(), dir.native(), gen);
+}
+
+sstables::shared_sstable new_env_sstable(sstables::test_env& env) {
+    testlog.debug("new_env_sstable: dir={}", env.tempdir().path());
+    return env.make_sstable(test_table_schema());
```

Comment:
The existing new_sstable() calls env.manager().make_sstable()
The new new_env_sstable() calls env.make_sstable() which, in turn, calls env.manager().make_sstable()

Is this helper really needed?